WalletWasabi/BranchNBound/Selector.cs

```diff
@@ -0,0 +1,28 @@
+using NBitcoin;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace WalletWasabi.BranchNBound
+{
+	public class Selector
```

Comment:
We won't need this base class, once we choose which algorithm to go with, right?